consensus/follower_test.go

```diff
@@ -159,6 +136,9 @@ func (s *HotStuffFollowerSuite) BeforeTest(suiteName, testName string) {
 func (s *HotStuffFollowerSuite) AfterTest(suiteName, testName string) {
 	s.cancel()
 	unittest.RequireCloseBefore(s.T(), s.follower.Done(), time.Second, "follower failed to stop")
+	s.notifier.AssertExpectations(s.T())
+	s.finalizer.AssertExpectations(s.T())
```

Comment:
```suggestion
```
This is called automatically since you are using `mockhotstuff.NewFinalizationConsumer(s.T())` to create mock

Reply:
💡 done in https://github.com/onflow/flow-go/pull/4234/commits/859cbc8f8b5a9a95447ee0d11c949d2e2de9612b